Added missing info for verify and clean schema feature (#6916)
umbraco/UmbracoDocs: 15/umbraco-deploy/deployment-workflow/deploy-dashboard.md

## Deploy Status

Here, you can check whether the latest deployment was successful or failed. You can also see the version of Umbraco Deploy in use and the timestamp of the last operation.

![Umbraco Deploy status](images/deploy-dashboard.png)

## Deploy Operations

With the Deploy operations, you can run different operations in Umbraco Deploy.

![Deploy operations](images/deploy-operations.png)

Below you can read what each operation will do when run through the dashboard.

### Update Umbraco Schema From Data Files

Running this operation will update the Umbraco Schema based on the information in the `.uda` files on disk.

### Verify and Clean Schema

> This operation deletes schema from your current environment if it does not have a matching UDA file. It functions as a batch operation for manually deleting each item in the Schema Comparison overview that has an exclamation mark in the 'File Exists' column.

Reviewer: ```suggestion
This operation deletes schema from your current environment if it does not have a matching UDA file. It manually deletes each item in the Schema Comparison overview with an exclamation mark in the 'File Exists' column.
```
